rfc8446.xml

```diff
@@ -1861,7 +1862,7 @@ completed handshake to change what groups they use in their
 "key_share" extension in subsequent connections.
 If the server has a group it prefers to the
 ones in the "key_share" extension but is still willing to accept the
-ClientHello, it SHOULD send "supported_groups" to update the
+ClientHello without sending a HelloRetryRequest, it SHOULD send "supported_groups" to update the
```

Comment:
This seems perhaps redundant, nbut OK.

Reply:
Yeah, since supported_groups can only be sent by the server in EE, this is probably redundant.

Reply:
Removed.